protocols/bleepsdao/overview.md

```diff
@@ -0,0 +1,34 @@
+# Bleeps DAO
+
+## Overview
+
+### What are Bleeps ?
+The very first sounds of the EVM
+
+![](https://i.imgur.com/QnoImbP.png)
+
+Bleeps use the same pioneer concept introduced in Mandalas.eth, where the metadata is fully generated from the contract, zero externalities, no backend, no ipfs, nor any client code required. While mandalas generated bitmaps, Bleeps generate sounds!
+
+### How to get a Bleep ?
+
+Bleeps are initially mintable through a prelimary sale at 0.1 ETH each Bleep. 75% of the proceeds go to the Bleeps DAO. There is a hard-coded max supply of 1024 sounds but only 576 sounds are currently available made of 9 instrument each composed of 64 notes (frequencies). 128 of them are currently reserved by the Bleeps' creator (wighawag) and might be distributed differently. The rest of the Bleeps (448) do not yet exist but can potentially be created later at the discretion of the DAO.
+
+### What are the benefits of owning a Bleep ?
+Owning a Bleep transforms a mere user into a Bleeper. Bleepers are given vote in the Bleeps DAO and earn a passive revenue from Melodies created from Bleeps they own. They also get to vote on proposal about the DAO treasury which they manage. One Bleep, One Vote.
+
+### Wait... Melodies ?
```

Comment:
New line

Reply:
Demo of markdown formatting https://hackmd.io/JSKzs_V0R--TP7bmHstn0w